### PR TITLE
adding kernel stress in the coverage

### PR DESCRIPTION
**Pull request by julpark-rh**
# Description

new suites for adding to the pipeline for xfstest and fsstress

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/pacific/cephfs/tier-1_fs_kernel_pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+  - test:
+      name: cephfs-mdsfailover-pinning-io
+      module: dir_pinning.py
+      config:
+        num_of_dirs: 200
+      polarion-id: CEPH-11227
+      desc: MDSfailover on active-active mdss,performing client IOs with no pinning at the first,later pin 10 dirs with IOs
+      abort-on-fail: false
+  - test:
+      name: cephfs subvolume authorize test
```
Comment (sumabai) [inline suites/pacific/cephfs/tier-1_fs_kernel_pipeline.yaml:201]: We are also running additional regression tests here that is already part other regression suites. Please help understand why we are repeating these tests in this suite?

Comment (julpark-rh) [reply suites/pacific/cephfs/tier-1_fs_kernel_pipeline.yaml:201]: i agree with you. I will talk to Neha with details.

**Comment by hkadam134** [comment]
https://github.com/red-hat-storage/cephci/blob/master/suites/pacific/cephfs/tier-0_fs_kernel.yaml
https://github.com/red-hat-storage/cephci/blob/master/suites/pacific/cephfs/tier-1_fs_kernel.yaml

There are already existing suites in Pacific for kernel - why don't we reuse the same ?

File: suites/pacific/cephfs/tier-1_fs_kernel_pipeline.yaml (hunk 1 above)
Comment (neha-gangadhar) [reply suites/pacific/cephfs/tier-1_fs_kernel_pipeline.yaml:201]: Julius, As we discussed we need not repeat the tests in the kernel suite. Lets keep the exclusive tests only. The other tests are covered in regression runs.

**Review by neha-gangadhar — APPROVED** [review]
lgtm

**Pull request merged**